website/showcase.json

```diff
@@ -1,362 +1,291 @@
-    "pinned": false
-  },
-  {
-    "name": "JD（手机京东）",
-    "icon": "jdcom.png",
-    "linkAppStore": "https://itunes.apple.com/cn/app/shou-ji-jing-dong-xin-ren/id414245413?mt=8",
-    "linkPlayStore": "https://app.jd.com/android.html",
-    "infoLink": "http://ir.jd.com/phoenix.zhtml?c=253315&p=irol-homeProfile",
-    "infoTitle": "JD.com is China’s largest ecommerce company by revenue and a member of the Fortune Global 500.",
-    "pinned": false
-  },
-  {
-    "name": "SoundCloud Pulse",
-    "icon": "soundcloud_pulse.jpg",
-    "linkAppStore": "https://itunes.apple.com/us/app/soundcloud-pulse-for-creators/id1074278256?mt=8",
-    "infoLink": "https://developers.soundcloud.com/blog/react-native-at-soundcloud",
-    "infoTitle": "Why React Native worked well for us",
-    "pinned": false
-  },
-  {
-    "name": "Tencent QQ",
-    "icon": "qq.png",
-    "linkPlayStore": "http://android.myapp.com/myapp/detail.htm?apkName=com.tencent.mobileqq",
-    "infoLink": "https://www.tencent.com/en-us/system.html",
-    "infoTitle": "QQ is China's largest messaging platform, with over 829 million active accounts",
-    "pinned": true
-  },
-  {
-    "name": "Townske",
-    "icon": "townske.png",
-    "linkAppStore": "https://itunes.apple.com/us/app/townske-stunning-city-guides/id1018136179?ls=1&mt=8",
-    "infoLink": "https://hackernoon.com/townske-app-in-react-native-6ad557de7a7c",
-    "infoTitle": "The experience of a web developer building an app using React Native",
-    "pinned": false
-  },
-  {
-    "name": "Vogue",
-    "icon": "vogue.jpeg",
-    "linkAppStore": "https://itunes.apple.com/app/apple-store/id1087973225?pt=45076&ct=site-promo&mt=8",
-    "infoLink": "http://www.vogue.com/app",
-    "infoTitle": "",
-    "pinned": false
-  },
-  {
-    "name": "adidas GLITCH",
-    "icon": "glitch.png",
-    "linkAppStore": "https://itunes.apple.com/gb/app/adidas-glitch-interchangeable/id1166181522?mt=8&cm_sp=FOOTBALL-_-UNDEFINED-_-CTA-DOWNLOAD-IPHONE",
-    "linkPlayStore": "https://play.google.com/store/apps/details?id=com.adidas.glitch&cm_sp=FOOTBALL-_-UNDEFINED-_-CTA-DOWNLOAD-ANDROID",
-    "infoLink": "https://medium.com/possible-cee/how-we-have-been-breaking-patterns-with-the-adidas-glitch-d734340fd40e",
-    "infoTitle": "Building Adidas GLITCH app with React Native",
-    "pinned": false
-  }
-]
+{
+  "meta": [
+    {
+      "name": "Facebook",
+      "icon": "facebook.webp",
+      "linkAppStore": "https://apps.apple.com/app/facebook/id284882215",
+      "linkPlayStore": "https://play.google.com/store/apps/details?id=com.facebook.katana",
+      "pinned": true
+    },
+    {
+      "name": "Facebook Ads Manager",
+      "icon": "adsmanager.png",
+      "linkAppStore": "https://apps.apple.com/us/app/facebook-ads-manager/id964397083",
+      "linkPlayStore": "https://play.google.com/store/apps/details?id=com.facebook.adsmanager",
+      "pinned": true
+    },
+    {
+      "name": "Instagram",
```

Comment:
Let's remove the Instagram app as there isn't active React Native usage there

Reply:
It's done! 🙂